### ISSUE TITLE
With an array of objects, the delete button always deletes the last object, regardless of what was selected

### ISSUE THREAD

**Issue by chrisjohnson** [eclipsesource/jsonforms#1128]
Pretty much as the title states. Using this schema:

```json
{"$schema": "http://json-schema.org/draft-04/schema#",
  "type": "array",
  "items":
  {"type": "object",
    "required": ["message"],
    "properties": {
      "message": {"type": "string"}
    }
  }
}
```

And no uischema, when adding 2 elements, checking the first element, and hitting delete, the last element is deleted. When adding 3 elements and checking the first two, the last two are deleted.

**Used Setup (please complete the following information):**
 - Framework: react
 - RendererSet: material

**Comment by edgarmueller** [comment]
Thanks for the report, we'll take care of it.

**Comment by edgarmueller** [comment]
The was fixed via #1146 and is part of 2.0.12-rc.3
